doc/llvm_obfuscation_tutorial.rst

```diff
@@ -505,6 +523,8 @@ To prevent overflowing we have set the maximum for Any1 and Any2 to 10, but this
 We need to make sure that x and y are not too big. The trick is that we have no information on their value at compile time.
 The solution we chose is to apply a bitmask to x and y in order to obtain a variable of which we know the max value.
 
+The careful reader may have noticed that randomly picking from ``IntegerVect`` has we did not check for uniqueness of its elements ;-)
```

Comment:
typo I guess 'has' -> 'as'.
But I don't understand your sentence, is there a part missing?